driver/bkimage/bkimage.go

```diff
@@ -0,0 +1,5 @@
+package bkimage
+
+const (
+	DefaultImage = "moby/buildkit:buildx-stable-1" // TODO: make this verified
```

Comment:
@tonistiigi Is this different from `moby/buildkit:v0.6.2`?

Reply:
It is atm, but it is a different tag in case we need to break compatibility in the future. Eg. `moby/buildkit:v0.7.0` might be equal to `moby/buildkit:buildx-stable-1` or `moby/buildkit:buildx-stable-2` depending if it is compatible.